docs/content/interop.md

```diff
@@ -25,83 +25,27 @@ You need to use the ICG to generate `.schema` files and type bindings for any ne
 
-```
-[InteropCodeGen.Settings]
-OutputPath=ExampleGame/Generated/
+### Example
 
-[InteropCodeGen.ClassesToGenerate]
-;ClassName=include_path.h
-;Leave empty if no includes required.
+The image below demonstrate how we setup the class `StarterProjectPlayerController` from the [Unreal GDK Starter Project’s](https://github.com/spatialos/UnrealGDKStarterProject), which depends on the include files `StarterProjectPlayerController.h`, `Camera/CameraAnim.h`, `Camera/CameraShake.h`, `GameFramework/HUD.h`, `GameFramework/LocalMessage.h` and `Particles/EmitterCameraLensEffectBase.h`.
 
-;MyClass
-/Script/ExampleGame.ExampleGameMyClass=ExampleGameMyClass.h
-/Script/ExampleGame.ExampleGameMyClass=IncludePath/MyClassDependency.h
-```
-
-### Example file description
-The  `DefaultEditorSpatialGDK.ini` file has two sections:
-* `InteropCodeGen.Settings`
-* `InteropCodeGen.ClassesToGenerate`
-
-**`InteropCodeGen.Settings`**<br/>
-In the  `InteropCodeGen.Settings` section you specify where the generated code goes.<br/>
-Make sure you list the `OutputPath=` relative to the `source` directory of your game.
-
-
-So, in the example above:
-
-```
-[InteropCodeGen.Settings]
-OutputPath=ExampleGame/Generated/
-```
-
-**`InteropCodeGen.ClassesToGenerate`**<br/>
-The `InteropCodeGen.ClassesToGenerate` section contains a list of the classes you want the ICG to create `.schema` and type bindings for.
-
-**Note:** You need to specify your class with the full PathName, which can be aqcuired by opening the context menu item **Copy reference** of the class in the **Content Browser** within the unreal editor.
-
-The section also contains the dependencies these classes require. These dependencies are an `#include` which the ICG adds to its generated code, so you need to make sure you list all the dependencies the classes need.
-
-In the example above, this is:
-
-```
-;MyClass
-/Script/ExampleGame.ExampleGameMyClass=ExampleGameMyClass.h
-/Script/ExampleGame.ExampleGameMyClass=Dependencies/MyClassDependency.h
-```
-
-The [Unreal GDK Starter Project’s](https://github.com/spatialos/UnrealGDKStarterProject)  `DefaultEditorSpatialGDK.ini` file contains the following:<br/>
-
-```
-;PlayerController
-/Script/StarterProject.StarterProjectPlayerController=StarterProjectPlayerController.h
-/Script/StarterProject.StarterProjectPlayerController=Camera/CameraAnim.h
-/Script/StarterProject.StarterProjectPlayerController=Camera/CameraShake.h
-/Script/StarterProject.StarterProjectPlayerController=GameFramework/HUD.h
-/Script/StarterProject.StarterProjectPlayerController=GameFramework/LocalMessage.h
-/Script/StarterProject.StarterProjectPlayerController=Particles/EmitterCameraLensEffectBase.h
-
-;PlayerCharacter
-/Script/StarterProject.PlayerCharacter=Weapons/Weapon.h
-/Script/StarterProject.PlayerCharacter=Items/Item.h
-```
+![Interop Codegen Settings](../assets/screen_grabs/toolbar/ICGSettings.png)
```

Comment:
```
![Interop Code Generator Settings](../assets/screen_grabs/toolbar/ICGSettings.png)
_Set up of the `StarterProjectPlayerController` class in the Starter Project_


```
We don't use "Codegen" in the  ICG's name.
Added an image caption as it helps people glancing through docs.